sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="m-3">
+    <v-row>
+      <header-title-value
+        title="Submitted date"
+        :value="
+          headerDetails.submittedDate
+            ? dateOnlyLongString(headerDetails.submittedDate)
+            : '-'
+        "
+      />
+      <Separator v-if="headerDetails.institutionName"></Separator>
```

Comment:
I am sorry, I didn't follow, why didn't we use the existing `vertical-divider`?

Reply:
We had "vertical dividers" in different ways in different components.
We do have a `.vertical-divider` CSS and now we have a component that can replace any other way that we did it before.
The `DetailsHeader.vue` can also use this component now.

Reply:
Changed it as a part of comment: https://github.com/bcgov/SIMS/pull/2297#discussion_r1336295198